app/constants/import_constants.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module ImportConstants
+  IMPORT_MAPPING = {
+    srg_id: 'SRGID',
+    stig_id: 'STIG ID',
+    rule_severity: 'Severity',
+    title: 'Requirement',
+    vuln_discussion: 'VulDiscussion',
```

Comment:
this field seems to be missing from the template examples and is mispelled in the `eat-tomcat` example file as `VulDiscusion`

Reply:
With regards to the typo, that is something that I think we'll have to just have them fix on their end. Since Vuln Discussion is missing from the templates this is likely something that can be made non-required.